ethsigner/signer/multikey/src/main/java/tech/pegasys/ethsigner/signer/multikey/MultiKeyTransactionSignerProvider.java

```diff
@@ -122,8 +121,9 @@ public TransactionSigner createSigner(final FileBasedSigningMetadataFile metadat
   }
 
   private boolean filenameMatchesSigningAddress(
-      final String signerAddress, final SigningMetadataFile metadataFile) {
+      final TransactionSigner signer, final SigningMetadataFile metadataFile) {
 
+    final String signerAddress = signer.getAddress().substring(2); // strip leading 0x
     if (!metadataFile.getBaseFilename().endsWith(signerAddress)) {
```

Comment:
Think this match should also the be case insensitive so that check summed Ethereum accounts can be used.

Reply:
done